install_dev.py

```diff
@@ -0,0 +1,255 @@
+import os
+import sys
+import shutil
+import subprocess
+import logging
+import tempfile
+import argparse
+logging.basicConfig()
+
```

Comment:
can you add a doc string here explaining briefly what this script does?

Reply:
Will add something. I do feel it's already 'documented': it's an interactive script (try running it with no options -  it will always ask before doing anything), optional command line arguments are also explained by `python install_dev.py --help` (as is standard), and comments in the code explain which each part does.

Reply:
then just write "Interactive script to install NuRadioMC/Reco and its dependencies, for more information call `python install_dev.py --help`"
 ;-)

Reply:
Added a docstring, which can be displayed using
```
import install_dev
help(install_dev)
```